CONTRIBUTING.md

```diff
@@ -201,6 +201,13 @@ And then in order to use that dylib in the tests, run the tests like this:
 MIRRORD_TEST_USE_EXISTING_LIB=../../target/universal-apple-darwin/debug/libmirrord_layer.dylib cargo test -p mirrord-layer
 ```
 
+On Apple Silicon, set `MIRRORD_MACOS_ARM64_LIBRARY` additionally to use the arm64 layer lib as well:
+```bash
+MIRRORD_TEST_USE_EXISTING_LIB=../../target/universal-apple-darwin/debug/libmirrord_layer.dylib \
+MIRRORD_MACOS_ARM64_LIBRARY=../../target/aarch64-apple-darwin/debug/libmirrord_layer.dylib \
+cargo test -p mirrord-layer
+```
```

Comment:
What do you think about deriving one var from the other if only one of them was specified?

Reply:
That would be a fairly reliable fallback, giving the `universal-apple-darwin` directory exists, the build fat mac script has most likely been run at this point. We can try deriving it and use it if the path exists, otherwise build the aarch64 target.

Reply:
Added deriving arm64 lib path from universal lib path as a fallback